layouts/partials/sidebar.html

```diff
@@ -0,0 +1,380 @@
+<nav class="sidebar__list">
+  <ul class="sidebar__list-items">
+    <li aria-expanded="false"
+      class="sidebar__category is-closed" data-category-name="minfraud">
+      <a class="sidebar__category-header" data-name="minfraud" tabindex="0">
+        <span class="sidebar__label">minFraud</span>
+      </a>
+      <ul class="sub-sidebar__items">
+        <li lass="sidebar__item">
```

Comment:
Something you may want to try: https://gohugo.io/content-management/menus/#define-in-site-configuration

This allows us to have a more concise sidebar.html file and replace the sidebar.ts file with hugo functions like [IsMenuCurrent](https://gohugo.io/methods/page/ismenucurrent/) and [CurrentSection](https://gohugo.io/methods/page/currentsection/)